internal/lambgofile/lambgofile.go

```diff
@@ -62,6 +68,8 @@ type Config struct {
 
 	OutDirectory   string   `yaml:"outDirectory"`
 	ZippedFileName string   `yaml:"zippedFileName"`
+	RawExtraFlags  string   `yaml:"extraFlags"`
```

Comment:
NP: Maybe call these `extraBuildFlags` or simply `buildFlags` for clarity, so it doesn't get confused with flags passed to Lambgo?